src/Dialogs/MapItemListSettingsPanel.cpp

```diff
@@ -37,15 +37,36 @@ MapItemListSettingsPanel::Prepare(ContainerWindow &parent, const PixelRect &rc)
 
   const MapSettings &settings = CommonInterface::GetMapSettings();
 
-  AddBoolean(_("Show Location row"),
+  AddBoolean(_("Lists Location row"),
```

Comment:
Why?

Reply:
Hm, the location row member of a list and in the next pull request, you can use it like a waypoint.

Reply:
Sentence no verb.